modules/javafx.graphics/src/test/java/test/com/sun/javafx/text/TextLayoutTest.java

```diff
@@ -95,6 +98,21 @@ private void verifyComplex(PrismTextLayout layout, boolean... complex) {
         }
     }
 
+    @Ignore() // ignored since StubFontLoader used in tests return fonts with null resources
+    @Test public void utf16chars() {
```

Comment:
Since this is `@Ignore`d with no real way to make it a useful test, maybe it would be best to revert the changes to this file?